### ISSUE TITLE
[Bug][Android] SearchBar render error in 4.5-pre2 on API 28

### ISSUE THREAD

**Issue by davidbritch** [xamarin/Xamarin.Forms#9339]
### Description

`SearchBar` isn't rendering correctly on Android in 4.5-pre2. Here's how a `SearchBar` should render on Android:

![filter-multiple-views-large](https://user-images.githubusercontent.com/8092460/73263199-f00e7c80-41c7-11ea-843b-419e0370ae40.png)

Here's how a `SearchBar` is currently rendering with Forms 4.5-pre2:

![Screenshot_1580213410](https://user-images.githubusercontent.com/8092460/73263275-22b87500-41c8-11ea-8be5-48c356f3d15c.png)

### Steps to Reproduce

1. Run the attached sample on Android.
2. Browse to any of the EmptyView pages that do filtering.

### Expected Behavior

A `SearchBar` fully appears at the top of the page.

### Actual Behavior

A `SearchBar` partially appears, but lacks the magnify icon, and the x icon that should appear.

### Basic Information

- Version with issue: 4.5-pre2
- IDE: VSMac 8.4.2
- Platform Target Frameworks: 
  - Android: API 28

### Screenshots

Behold the `SearchBar`:

![Screenshot_1580213410](https://user-images.githubusercontent.com/8092460/73263031-88f0c800-41c7-11ea-8bb8-cb9fb0f5b59d.png)

### Reproduction Link

[CarouselViewDemos.zip](https://github.com/xamarin/Xamarin.Forms/files/4122257/CarouselViewDemos.zip)

**Comment by samhouts** [comment]
possibly related to #9266

**Comment by RedasP** [comment]
We're experiencing this issue on Xamarin.Forms 4.4.0.991477. Could you suggest a workaround?

**Comment by agat366** [comment]
Any hope for the workaround?..

**Comment by Hummel37** [comment]
> Any hope for the workaround?..

Yes, see https://github.com/xamarin/Xamarin.Forms/issues/9266#issuecomment-619115611
